webapp/sources/rudder/rudder-rest/src/test/scala/com/normation/rudder/rest/RestTestSetUp.scala

```diff
@@ -978,26 +979,6 @@ class RestTestSetUp {
 
   val baseTempDirectory = mockGitRepo.abstractRoot
 
-  /*
-   * We will commit some revisions for packageManagement technique:
-   * - init commit: what is on repos
-   * - commit 1: change metadata section (impact on directive details, but not on variables)
-   * - commit 2: revert to initial state
-   */
-  def updatePackageManagementRevision(): Unit = {
-    val metadata = mockGitRepo.configurationRepositoryRoot / "techniques/applications/packageManagement/1.0/metadata.xml"
-
-    val orig = metadata.contentAsString(StandardCharsets.UTF_8)
-    val mod  = orig.replaceAll("""name="Package version" """, """name="AN OTHER REVISION OF TECHNIQUE PACKAGE MANAGEMENT" """)
-
-    metadata.write(mod)
-    mockGitRepo.gitRepo.git.add().setUpdate(true).addFilepattern(".").call()
-    mockGitRepo.gitRepo.git.commit().setMessage("new revision of packageManagement/1.0 technique").call()
-    metadata.write(orig)
-    mockGitRepo.gitRepo.git.add().setUpdate(true).addFilepattern(".").call()
-    mockGitRepo.gitRepo.git.commit().setMessage("revert to original content for packageManagement/1.0 technique").call()
-  }
-
```

Comment:
Removed because unused and caused a problem with commit version (exhibited by zio-test init order ?)